test/unit/messaging/index.test.js

```diff
@@ -0,0 +1,13 @@
+jest.mock('ffc-messaging')
+jest.mock('../../../app/data')
+const messageService = require('../../../app/messaging')
+
+describe('messaging', () => {
+  afterAll(async () => {
+    await messageService.stop()
+  })
+
+  test('runs', async () => {
+    await messageService.start()
```

Comment:
does this test need an `expect`?

Reply:
It doesn't as there's nothing to validate.  

The test will therefore pass if the starting the service does not throw an error.

To be honest, I'm not really a fan of a test like this but given there is no actual behaviour in this application yet, there's not a lot to really test happens.